figures/permissions.py

```diff
@@ -46,14 +47,13 @@ def is_site_admin_user(request):
         if figures.helpers.is_multisite():
             if request.user.is_active:
                 current_site = django.contrib.sites.shortcuts.get_current_site(request)
-                org_ids = Organization.objects.filter(
-                    sites__in=[current_site]).values_list('id',
-                                                          flat=True)
-                return UserOrganizationMapping.objects.filter(
-                    organization_id__in=org_ids,
-                    user=request.user,
-                    is_active=True,
-                    is_amc_admin=True).exists()
+                edx_organization_ids = Organization.objects.filter(
+                    edlysuborganization__lms_site=current_site
+                ).values_list('id', flat=True)
+                return EdlyUserProfile.objects.filter(
+                    edly_sub_organizations__edx_organization__in=edx_organization_ids,
+                    user=request.user
```

Comment:
@zubair-arbi the condition in appsembler's code seems to be using some kind of admin check as well. Should I place some admin check as well?